### PR TITLE
Add readme text for Parameterized Configs

### PR DESCRIPTION
**Pull request by divyekapoor**
We dug into the code internally to figure out how the ParameterizedConfigs work. 
Updating the Readme to make this simpler. 

(Open to revisions / fixes. If any changes do need to be made, could we please merge this PR and fix forward?)

### CODE REVIEW

**Comment by divyekapoor** [comment]
Seems like this can be edited by maintainers before merging. Please go ahead with any changes needed!

**Review by brianm — APPROVED** [review]

**Pull request merged**
